src/main/java/org/gridsuite/shortcircuit/server/utils/FaultResultSpecificationBuilder.java

```diff
@@ -28,8 +32,27 @@ public static Specification<FaultResultEntity> resultUuidEquals(UUID value) {
     }
 
     public static Specification<FaultResultEntity> buildSpecification(UUID resultUuid, List<ResourceFilter> resourceFilters) {
-        Specification<FaultResultEntity> specification = Specification.where(resultUuidEquals(resultUuid));
-        return SpecificationUtils.appendFiltersToSpecification(specification, resourceFilters);
+        return (final Root<FaultResultEntity> root, final CriteriaQuery<?> query, final CriteriaBuilder builder) -> {
+            query.distinct(true);
+
+            Specification<FaultResultEntity> specification = Specification.where(resultUuidEquals(resultUuid));
+            List<ResourceFilter> parentsFilters = resourceFilters.stream()
+                    .map(filter -> {
+                        if (FeederResultEntity.Fields.connectableId.equals(filter.column())) {
```

Comment:
I'm not sure to understand, how can we keep filter on `connectableId` for both parent and children ? What's the point to filter on the feeders if there is no fault containing them ?

Reply:
If I understand correctly, the point is to filter out parents which would have empty children with filters passed as params

Reply:
That was it, but it is not needed anymore and removed.

Reply:
I missunderstood how it works